pom.xml

```diff
@@ -88,6 +88,8 @@
         <spark.version>2.2.0</spark.version>
         <scala.version>2.11.6</scala.version>
         <netty-all.version>4.0.29.Final</netty-all.version>
+        <spark.warehouse>-Dsplice.spark.sql.warehouse.dir=${project.build.directory}/spark-warehouse</spark.warehouse>
+        <hbase.rootdir>-Dhbase.rootdir=${project.build.directory}/hbase</hbase.rootdir>
```

Comment:
Would this break the "secure" profile, or would it just override these properties? I'm not even sure it works right now, but it would be nice testing it.

Reply:
These are now default settings used outside secure/insecure profiles.
The _secure_ profile overrides them. The _insecure_ profile relies on the default values.

Reply:
Got it, thanks!